packages/thirdweb/src/contract/actions/resolve-abi.ts

```diff
@@ -227,6 +227,78 @@ const BASE_ROUTER_ABI = {
   type: "function",
 } as const;
 
+const CORE_CONTRACT_ABI = {
```

Comment:
just making sure: we absolutely *need* this abi here statically and every element in it?

can't use extensions for this for example?

Reply:
Removed this abi.
Using `getInstalledExtensions` from generated read functions instead.